src/components/Microsoft.DotNet.UpgradeAssistant.MSBuild/MSBuildProject.cs

```diff
@@ -90,10 +90,26 @@ private ProjectOutputType GetDefaultOutputType()
         public ValueTask<ProjectComponents> GetComponentsAsync(CancellationToken token)
             => _componentIdentifier.GetComponentsAsync(this, token);
 
+        public IEnumerable<string> FindFiles(ProjectItemMatcher matcher)
+        {
+            // TODO: code review - do we want to just flip the parameter order and make the itemType an optional parameter?
```

Comment:
I'm for this. The matcher will always be needed, but the project type is not

Reply:
Agreed

Reply:
Will flip